packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.scss

```diff
@@ -0,0 +1,42 @@
+.json-schema-renderer {
+
+	margin: 20px 0 0 20px;
```

Comment:
Any way to use one margin value instead? 
There are some variables for that in theme package.